### PR TITLE
docs: Updated README for the ts-apollo-mongodb-datasync-backend

### PR DESCRIPTION
**Pull request by RinkiyaKeDad**
Let me know what to improve :)

### CODE REVIEW

File: templates/ts-apollo-mongodb-datasync-backend/README.md [hunk 1]
```diff
@@ -1,45 +1,18 @@
-## Graphback Apollo+MongoDB Data Synchronization Runtime Template
+## Graphback MongoDB with Datasync + Apollo Server Template
 
-Template showcases Graphback Data Synchronization capabilities using
-resolver layer created in-memory as opposed to codegeneration.
+A template that provides you with an easy setup for your application's backend using MongoDB and Apollo server with TypeScript.
```
Comment (craicoverflow) [inline templates/ts-apollo-mongodb-datasync-backend/README.md:3]: Description here mentions nothing of Data Sync capabilities of this template - would be nice to have a line showcasing this.

**Review by craicoverflow — CHANGES_REQUESTED** [review]
Left one comment - just need a short line mentioning data sync capability of this template. I know it is further down, but some people will not make it that far when reading.

**Comment by RinkiyaKeDad** [comment]
Please check now @craicoverflow.

**Review by craicoverflow — APPROVED** [review]
Fantastic once again!

**Pull request merged**